doc/radosgw/admin.rst

```diff
@@ -395,9 +395,9 @@ following form:
 Quota Management
 ================
 
-The Ceph Object Gateway enables you to set quotas on users and buckets owned by
-users. Quotas include the maximum number of objects in a bucket and the maximum
-storage size a bucket can hold.
+The Ceph Object Gateway makes it possible for you to set quotas on users and
```

Comment:
"makes it possible" seems a little dramatic, suggest 
>Quotas may be set for The Ceph Object Gateway on users and their buckets.

Reply:
I like that better. I've never thought of "makes it possible" as a dramatic clause, but I see your point now. It does have a "You've never really peeled a potato until you've peeled a potato with Tater Mitts (TM)!" quality.

Reply:
Explore the possibilities.   Make it real.  Carpe diem.

https://g.co/kgs/qJxpmcd

It's kinda like when I went to New Zealand and would see camper van rental sites saying "Bookings *essential*"

I stayed up too late so I'm kinda punchy.

Reply:
Accepted.